<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Choose</class>
 <widget class="QWidget" name="Choose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>130</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>员工姓名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>781</width>
     <height>202</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>40</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>163</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>163</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{color:white;background-color: rgb(68, 113, 196);border-radius:5px}
QPushButton::pressed{background-color: rgb(142, 170, 220);}</string>
      </property>
      <property name="text">
       <string>磁辊检测</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>163</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>163</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{color:white;background-color: rgb(68, 113, 196);border-radius:5px}
QPushButton::pressed{background-color: rgb(142, 170, 220);}</string>
      </property>
      <property name="text">
       <string>结果统计</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="minimumSize">
       <size>
        <width>163</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>163</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{color:white;background-color: rgb(68, 113, 196);border-radius:5px}
QPushButton::pressed{background-color: rgb(142, 170, 220);}</string>
      </property>
      <property name="text">
       <string>驱动参数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{color:white;background-color: rgb(68, 113, 196);border-radius:5px}
QPushButton::pressed{background-color: rgb(142, 170, 220);}</string>
      </property>
      <property name="text">
       <string>选取图片处理</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>70</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{color:white;background-color: rgb(68, 113, 196);border-radius:5px}                               QPushButton::pressed{background-color: rgb(142, 170, 220);}</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>570</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="text">
    <string>版本信息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>570</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="text">
    <string>当前版本</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>570</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>567</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{color:white;background-color: rgb(68, 113, 196);border-radius:1px}                               QPushButton::pressed{background-color: rgb(142, 170, 220);}</string>
   </property>
   <property name="text">
    <string>检查更新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
